Sources/_StringProcessing/RegexDSL/Core.swift

```diff
@@ -131,7 +125,7 @@ extension RegexProtocol {
         let typeErasedMatch = captures.matchValue(
           withWholeMatch: input[range]
         )
-        convertedMatch = _openExistential(typeErasedMatch, do: bitCastToMatch)
+        convertedMatch = typeErasedMatch as! Match
```

Comment:
(out of curiosity) Is this from this change, or is this from the proper use of tuples?

Reply:
>  is this from the proper use of tuples?

Since we are no longer using nominal tuples, this makes it a safer than a bitcast.  It should have been done in #127 but I just came across this yesterday.